Sprint-3/revise/implement/count.test.js

```diff
@@ -15,3 +15,96 @@
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+//---------------------------------------------------------------------------------------------------
```

Comment:
😵 This is so much code! I definitely think it would make sense to stop and check before writing epics like this.

Go test first, writing the simplest test you can think of, that satisfies the simplest condition defined in the criteria. Just write one test and then just enough code to pass that test. 

Spend the time thinking of the simplest thing that can possibly work.

Reply:
I've simplified the code and focused on writing smaller, simpler tests first, as suggested. I'll continue using this approach going forward. 

Thank you @SallyMcGrath  for your valuable feedback.